README.md

```diff
@@ -196,12 +196,15 @@ There are currently two release channels for binaries, `stable` and `nightly`.
 The `nightly` channel has releases built after each push to the master branch, and will have the most recent fixes and additions, but also have more risk of regressions. They are available in [their own repo](https://github.com/yt-dlp/yt-dlp-nightly-builds/releases).
 
 When using `--update`/`-U`, a release binary will only update to its current channel.
-This release channel can be changed by using the `--update-to` option. `--update-to` can also be used to upgrade or downgrade to specific tags from a channel.
+This release channel can be changed by using the `--update-to` option. `--update-to` can also be used to upgrade or downgrade to specific tags from a channel. When using `--update-to`, it will only update if the new version is higher than the current version. This will be the case even when changing channels.
```

Comment:
> When using `--update-to`, it will only update if the new version is higher than the current version

This is only true when tag is not specified, correct?

Reply:
Yes, `yt-dlp --update-to "stable@2023.04.02.2"` worked in a local test with [Grub4K/yt-dlp@2023.04.03](https://github.com/Grub4K/yt-dlp/releases/tag/2023.04.03). Maybe the documentation can be made more clear by flipping the two sentences around?
> This release channel can be changed by using the `--update-to` option. When using `--update-to`, it will only update if the new version is higher than the current version. This will be the case even when changing channels. `--update-to` can be used to upgrade to specific tags from a channel, which will even work for downgrading.

Reply:
How about
```suggestion
`--update-to CHANNEL` can be used to switch to a different channel when a newer version is available. `--update-to [CHANNEL@]TAG` can also be used to upgrade or downgrade to specific tags from a channel.
```

Reply:
Thats shorter and more concise. Should the `options.py` be adjusted as well or is that fine as is?

Reply:
That looks fine to me.